File: sample-conference-videochat/sample-conference-videochat/Components/Users/Views/UserTableViewCell/UserTableViewCell.m
//
//  UserTableViewCell.m
//  LoginComponent
//
//  Created by Andrey Ivanov on 03/06/16.
//  Copyright © 2016 Quickblox. All rights reserved.
//

#import "UserTableViewCell.h"
#import "CheckView.h"

@interface UserTableViewCell()

@property (weak, nonatomic) IBOutlet CheckView *checkView;
@property (weak, nonatomic) IBOutlet UILabel *fullNameLabel;
@property (weak, nonatomic) IBOutlet UIImageView *userImageView;

@end

@implementation UserTableViewCell

#pragma mark - Setters

- (void)setFullName:(NSString *)fullName {
    
    self.fullNameLabel.text = fullName;
}

- (void)setCheck:(BOOL)isCheck {
    
    self.checkView.check = isCheck;
}

- (void)setUserImage:(UIImage *)image {
    
    self.userImageView.image = image;
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/Users/Views/UserTableViewCell/UserTableViewCell.swift
//
//  UserTableViewCell.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 12/10/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class UserTableViewCell: UITableViewCell {
    //MARK: - IBOutlets
    @IBOutlet private weak var checkView: CheckView!
    @IBOutlet private weak var fullNameLabel: UILabel!
    @IBOutlet private weak var userImageView: UIImageView!
    
    //MARK: - Properties
    var fullName: String? {
        didSet {
            fullNameLabel.text = fullName
        }
    }
    
    var check: Bool? {
        didSet {
            checkView.check = check
        }
    }
    
    var userImage: UIImage? {
        didSet {
            userImageView.image = userImage
        }
    }
}
